pkgs/by-name/in/intel-compute-runtime/package.nix

```diff
@@ -34,11 +35,12 @@ stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
-    "-DSKIP_UNIT_TESTS=1"
-    "-DIGC_DIR=${intel-graphics-compiler}"
-    "-DOCL_ICD_VENDORDIR=${placeholder "out"}/etc/OpenCL/vendors"
+    (lib.cmakeBool "SKIP_UNIT_TESTS" true)
+    (lib.cmakeFeature "IGC_DIR" (builtins.toString intel-graphics-compiler))
+    (lib.cmakeFeature "OCL_ICD_VENDORDIR" "${placeholder "out"}/etc/OpenCL/vendors")
     # The install script assumes this path is relative to CMAKE_INSTALL_PREFIX
-    "-DCMAKE_INSTALL_LIBDIR=lib"
+    (lib.cmakeFeature "CMAKE_INSTALL_LIBDIR" "lib")
+    (lib.cmakeFeature "CMAKE_CXX_FLAGS" "-Wno-error=deprecated-declarations")
```

Comment:
Please patch out the Werror from here https://github.com/intel/compute-runtime/blob/d54b74e8bd072ce6a97a2e87faef0d2b9409dd4b/CMakeLists.txt#L854

I am not interested to chaise every warning with upstream.

Reply:
I don't understand what you're asking me to do.

Reply:
I assumed the build failed because of Werror in which case we should rather remove that as new compiler updates usually add new warnings which we then would need to fix :/